README.md

```diff
@@ -43,8 +43,36 @@ IntelliJ doesn't cope well with the processing performed in the maven overlay so
 
 Note in this example we've specified the following system properties use a configuration overrides file similar to what's used when the app is deployed.
 
-```
--Dgeonetwork.jeeves.configuration.overrides.file=/home/craigj/temp/geonetwork-build-overrides.xml
-```
+    -Dgeonetwork.jeeves.configuration.overrides.file=/home/craigj/temp/geonetwork-build-overrides.xml
+
+Its also possible to specify a separate location for the data directory used to store uploaded files, schema plugins
+the lucene index and other GeoNetwork state using the geonetwork.dir system property.  For example for my local
+IntelliJ setup I use:
 
+    -Dgeonetwork.dir=/home/craigj/temp/geonetwork-build-data-dir
+
+as well as the config overrides parameter above.  Be careful with this option though as
+changes to the data directory included in the war won't automatically get deployed to 
+a separate data directory.
+ 
 ![image](https://user-images.githubusercontent.com/1860215/121621720-0e6cd380-cab0-11eb-8b5f-d57632f82fe6.png)
+
+### Connecting to postgres
+
+To connect to a postgres database instead of the default H2 one, use a configuration overrides file that 
+imports the spring jndi configuration:
+
+    <spring>
+        <import file="/WEB-INF/config-db/jndi-postgres-postgis.xml"/>
+    </spring>
+
+A configuration overrides that does just that can be found at [jndi-override.xml](jndi-override.xml).
+
+The jndi connection details will then need to be added to the context for this application. A sample context.xml
+file to do this can be found at [context.xml example](main/src/main/webapp/META-INF/context_sample.xml).
+Copy this file to  main/src/main/webapp/META-INF/context.xml and change the connection details to point to
+your postgres database.
+
+To use this configuration add the location of the configuration overrides file to your startup options e.g.
+
+    -Dgeonetwork.dir={geonetwork-build directory}/jndi-override.xml
```

Comment:
is this correct? should it be something other than `geonetwork.dir`

Reply:
Right you are.  Fixed